operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -0,0 +1,259 @@
+# Documentation Style Guide
+
+When writing for Mattermost, always think about the audience that will be reading your writing. Mattermost’s 
+audience can vary, but we typically communicate with a technical audience and want to present Mattermost as 
+being a friend of the practitioner in development, IT, and operations. It must be accurate and clear, be 
+presented with a style and tone that is appropriate for technical content, and easy to navigate.
+
+## Document structure
+
+Structure and organization are an important part of a document's ease of use and its understandability. 
+Information should be organized and presented in a logical order, with similar subjects grouped together in the 
+same section. In most cases, a document has a title, an introductory paragraph, and one or more sections.
+
+Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are 
+easier to write and edit, and are easier to translate.
+
+### Document sections
+
+To make pages easier for people to quickly scan for the content that they're looking for, break your 
+document up into logical sections. Each section should have a title, and the title should relate 
+to the content of the section. Use sentence case for the section title.
+
+A section title is not required if you have only one section.
+
+### Document title
+
+Use a title that accurately reflects the content of the document. People scan the table of contents 
+looking for answers; it's often faster than using the built-in search engine.
+
+Use title case for document titles. For more information and an example of capitalization, see :ref:`capital`.
```

Comment:
Not sure how to check if the reference link works.

Reply:
Good point. I'd normally do a local build to check but realized that it doesn't work the same as the Sphinx docs. I'll remove for now.